Sources/StreamChat/Repositories/SyncRepository.swift

```diff
@@ -132,6 +132,11 @@ class SyncRepository {
         // Enter recovery mode so no other requests are triggered.
         apiClient.enterRecoveryMode()
 
+        // Run offline actions requests as the first thing
+        if config.isLocalStorageEnabled {
+            operations.append(ExecutePendingOfflineActions(offlineRequestsRepository: offlineRequestsRepository))
+        }
+        
```

Comment:
Moved it to the first item on the syncing list so that we retry more quickly rather than waiting after other stuff.